### PR TITLE
Bug/post refactor

### PR DESCRIPTION
**Pull request by AlexanderBoriskin**
See #249

### CODE REVIEW

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 1]
```diff
@@ -6,10 +6,10 @@
     <span slot="center">
       {{$t('navbar.messageUs')}}&nbsp;<a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
     </span>
-    <message-dropdown/>
-    <notification-dropdown/>
-    <language-dropdown/>
-    <profile-dropdown>
+    <message-dropdown @toggle-menu="hideSidebar()"/>
+    <notification-dropdown @toggle-menu="hideSidebar()"/>
+    <language-dropdown @toggle-menu="hideSidebar()"/>
```
Comment (smartapant) [inline src/components/admin/app-navbar/AppNavbar.vue]: after choosing language, dd is not getting closed

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
+    <profile-dropdown @toggle-menu="hideSidebar()">
```
Comment (smartapant) [inline src/components/admin/app-navbar/AppNavbar.vue]: onResize toggles sidebard different (desktop -> mobile, mobile -> desktop) than in vuestic.epicmax.co

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 3]
```diff
@@ -42,6 +42,11 @@
         required: true
       }
     },
+    methods: {
+      hideSidebar () {
```
Comment (smartapant) [inline src/components/admin/app-navbar/AppNavbar.vue]: this is strange a little, so many repetitive @toggleMenu events. what do you think @asvae?

File: src/components/admin/app-sidebar/components/SidebarLink.vue [hunk 4]
```diff
@@ -22,6 +23,13 @@
         default: '_self'
       }
     },
+    methods: {
+      toggleMenuItem () {
```
Comment (smartapant) [inline src/components/admin/app-sidebar/components/SidebarLink.vue]: shall it really be there? looks messy, @asvae please note

File: src/components/admin/app-sidebar/components/SidebarLink.vue [hunk 5]
```diff
@@ -22,6 +23,13 @@
...
+        if (document.documentElement.clientWidth < 992) {
```
Comment (asvae) [inline src/components/admin/app-sidebar/components/SidebarLink.vue]: What is that?

File: src/components/admin/AppLayout.vue [hunk 6]
```diff
@@ -16,6 +16,7 @@
   import VuesticLayout from '../../vuestic-theme/vuestic-components/vuestic-layout/VuesticLayout'
   import AppNavbar from './app-navbar/AppNavbar'
   import AppSidebar from './app-sidebar/AppSidebar'
+  import Layout from './../../vuestic-theme/vuestic-directives/Layout'
```
Comment (smartapant) [inline src/components/admin/AppLayout.vue]: vuestic-theme might be used without relative paths

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 7]
```diff
@@ -6,10 +6,10 @@
     <span slot="center">
       {{$t('navbar.messageUs')}}&nbsp;<a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
     </span>
-    <message-dropdown @toggle-menu="hideSidebar()"/>
-    <notification-dropdown @toggle-menu="hideSidebar()"/>
-    <language-dropdown @toggle-menu="hideSidebar()"/>
-    <profile-dropdown @toggle-menu="hideSidebar()">
+    <message-dropdown :isOpen.sync="valueProxy"/>
```
Comment (smartapant) [inline src/components/admin/app-navbar/AppNavbar.vue]: valueProxy - variable name is not descriptive

File: src/vuestic-theme/vuestic-directives/Layout.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+          selector.classList.toggle('i-menu-expanded')
+          selector.classList.toggle('i-menu-collapsed')
+        }
+      }, 0)
+    })
+
+    sidebar.addEventListener('click', () => {
+      if (document.documentElement.clientWidth < 992) {
+        sidebar.classList.add('sidebar-hidden')
+      }
```
Comment (smartapant) [inline src/vuestic-theme/vuestic-directives/Layout.js]: add `unbind` metod (https://vuejs.org/v2/guide/custom-directive.html)

File: src/vuestic-theme/vuestic-directives/Layout.js [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+  bind: function (el, binding) {
+    let navbar = el.querySelector('.vuestic-navbar')
+    let sidebar = el.querySelector('.vuestic-sidebar')
+
+    navbar.addEventListener('click', (evt) => {
+      setTimeout(function () {
+        let openedDropdown = navbar.querySelector('.show')
+        let selector = navbar.querySelector('.header-selector')
+        if (openedDropdown && document.documentElement.clientWidth < 992 &&
```
Comment (smartapant) [inline src/vuestic-theme/vuestic-directives/Layout.js]: save 992px as a const for now

**Pull request merged**